lib/mspec/commands/mspec.rb

```diff
@@ -107,7 +107,7 @@ def multi_exec(argv)
       IO.popen([env, *command], "rb+")
     }
 
-    puts children.map { |child| child.gets }.uniq
+    puts children.map { |child| child.gets("").chomp }.uniq
```

Comment:
The idea is nice, I might use it later or to be more resilient to the interpreter extra output, thanks.